<?xml version="1.0" encoding="UTF-8"?>
<alr:test-suite xmlns:alr="urn:model.allure.qatools.yandex.ru" start="1729890314574" stop="1729890314583" version="1.4.0">
  <name>Magento\Customer\Test\Unit\Controller\Section\LoadTest</name>
  <test-cases>
    <test-case start="1729890314575" stop="1729890314579" status="passed">
      <name>testExecute with data set #0</name>
    </test-case>
    <test-case start="1729890314580" stop="1729890314581" status="passed">
      <name>testExecute with data set #1</name>
    </test-case>
    <test-case start="1729890314581" stop="1729890314583" status="passed">
      <name>testExecuteWithThrowException</name>
    </test-case>
  </test-cases>
</alr:test-suite>
